Cart/Cart.html

```diff
@@ -6,142 +6,144 @@
                                 </div>
-                                <div class="card-container">
-                                    <div class="card-top-section">
-                                        <div class="image-box">
-                                            <img src="https://img.tatacliq.com/images/i7/658Wx734H/MP000000010664108_658Wx734H_202109211508401.jpeg"
-                                                alt="bag-image" />
-                                        </div>
-                                        <div class="card-details">
-                                            <h3>Michael Kors</h3>
-                                            <p>Hamilton Legacy Large Logo Belted Satchel</p>
-                                            <p>Brown<p>
-                                            <div class="quantity">
-                                                <label for="quantity">Qty
-                                                    <input class="cart-quantity" type="number" id="quantity" name="quantity" min="1" max="5"
-                                                        placeholder="1">
-                                            </div>
-                                            <div>₹ 22,500</div>
-                                            <div class="card-bottom-section-links">
-                                                <a href="#">Remove</a>
-                                                <span>|</span>
-                                                <a href="#">Move to wishlist</a>
-                                            </div>
-                                        </div>
-                                        <a href="#" class="card-closebtn">&times;</a>
+                            </div>
+                            <div class="card-container">
+                                <div class="card-top-section">
+                                    <div class="image-box">
+                                        <img src="https://img.tatacliq.com/images/i7/658Wx734H/MP000000010664108_658Wx734H_202109211508401.jpeg"
+                                            alt="bag-image" />
                                     </div>
-                                    <!-- <div class="img-tint"></div> -->
-
+                                    <div class="card-details">
+                                        <h3>Michael Kors</h3>
+                                        <p>Hamilton Legacy Large Logo Belted Satchel</p>
+                                        <p>Brown<p>
+                                                <div class="quantity">
+                                                    <label for="quantity">Qty
+                                                        <input class="cart-quantity" type="number" id="quantity"
+                                                            name="quantity" min="1" max="5" placeholder="1">
```

Comment:
A suggestion instead of nested divs, you can use semantic tags liks section, article etc.